package.json

```diff
@@ -32,7 +32,7 @@
     "@angular-eslint/eslint-plugin-template": "^17.2.0",
     "@angular-eslint/template-parser": "^17.2.0",
     "@angular/cli": "^17.0.8",
-    "@hadronous/pic": "0.2.1-b1",
+    "@hadronous/pic": "0.2.1-b2",
```

Comment:
I made a beta release of this with the updated dfinity packages. ic-angular is already on the newer versions so there was a conflict.